mysql-test/suite/wsrep/r/variables.result

```diff
@@ -66,6 +66,7 @@ wsrep_cert_interval	#
 wsrep_open_transactions	#
 wsrep_open_connections	#
 wsrep_incoming_addresses	#
+wsrep_debug_sync_waiters	#
```

Comment:
This is problematic test.

(1) We do not run tests using Debug Galera library on bb
(2) Test depends on certain Galera library version (at the moment we can't use it everywhere)

Question is does this even test anything meaningful or should we have some requirement for some Galera library version ?

Reply:
> (1) We do not run tests using Debug Galera library on bb

The test includes galera_have_debug_sync.inc. Shouldn't that be enough to skip the test?

> (2) Test depends on certain Galera library version (at the moment we can't use it everywhere)
> Question is does this even test anything meaningful or should we have some requirement for some Galera library version ?

Which part depends on specific version of galera? Occasionally, we have a new status variable in Galera library. When that happens, the test must be recorded again.
I don't know if this test is useful or not, I think it originally came from Nirbhay. I would suggest you remove the parts that depend on galera specific versions.